examples/get_started/json-csv-reader.py

```diff
@@ -45,54 +46,55 @@ def main():
         "gs://datachain-demo/openimages-v6-test-jsonpairs/08392c290ecc9d2a.json"
     )
     json_pairs_ds = DataChain.from_json(
-        uri, schema_from=schema_uri, jmespath="@", model_name="OpenImage"
+        uri, schema_from=schema_uri, jmespath="@", model_name="OpenImage", anon="True"
     )
     json_pairs_ds.show()
 
     uri = "gs://datachain-demo/coco2017/annotations_captions/"
 
     print()
     print("========================================================================")
-    print("Reading JSON schema from main COCO annotation")
+    print("Print JSON schema in Pydantic format from main COCO annotation")
```

Comment:
[Q] Do we need these print statements? Can they just be comments?

Reply:
tbh, not that important at this point .. i'm fine with either

Reply:
My gripe is that this is the only example with this style of `print`s in it